services/todo-legacy/controllers/todo-controller.api.ts

```diff
@@ -0,0 +1,225 @@
+              },
+              body: {
+                type: "string",
+                description: "The body of the Todo entry."
+              }
+            }
+          }
+        }
+      },
+      post: {
+        "x-operation-name": "createTodo",
+        parameters: [
+          {
+            name: "data",
+            in: "body",
+            description: "The Todo model instance.",
+            required: true,
+            format: "JSON"
+          }
+        ],
+        responses: {
+          201: {
+            schema: {
+              id: {
+                type: "number",
+                description: "The ID number of the Todo entry."
+              },
+              title: {
+                type: "string",
+                description: "The title of the Todo entry."
+              },
+              body: {
+                type: "string",
+                description: "The body of the Todo entry."
+              }
+            }
+          }
+        }
+      },
+      delete: {
+        "x-operation-name": "deleteTodo",
+        parameters: [
+          {
+            name: "title",
+            in: "query",
+            description:
+              "The criteria used to narrow down the number of customers returned.",
+            required: false,
+            type: "string",
+            format: "JSON"
+          }
+        ],
+        responses: {
+          204: {
+            description: "The resource has been deleted." 
+          }
+        }
+      }
+    },
+    "/todo/{id}": {
+      get: {
+        "x-operation-name": "getTodoById",
+        parameters: [
+          {
+            name: "id",
+            in: "path",
+            description:
+              "The criteria used to narrow down the number of customers returned.",
+            required: false,
+            type: "string",
+            format: "JSON"
+          }
+        ],
+        responses: {
+          200: {
+            schema: {
+              id: {
+                type: "number",
+                description: "The ID number of the Todo entry."
+              },
+              title: {
+                type: "string",
+                description: "The title of the Todo entry."
+              },
+              body: {
+                type: "string",
+                description: "The body of the Todo entry."
+              }
+            }
+          }
+        }
+      },
+      put: {
+        "x-operation-name": "replaceTodo",
+        parameters: [
+          {
+            name: "id",
+            in: "path",
+            description: "The todo ID.",
+            required: true,
+            type: "string",
+            format: "JSON"
+          },
+          {
+            name: "data",
+            in: "body",
+            description: "The Todo model instance.",
+            required: true,
+            format: "JSON"
+          }
+        ],
+        responses: {
+          202: {
```

Comment:
This does not look right to me.

> https://httpstatuses.com/202
> The request has been accepted for processing, but the processing has not been completed. The request might or might not eventually be acted upon, as it might be disallowed when processing actually takes place.

IMO, you should return `200`.